.github/workflows/update-e2e-template.yml

```diff
@@ -70,7 +70,7 @@ jobs:
       - name: Check for changes
         id: check_changes
         run: |
-          if git diff --quiet "${{ github.event.pull_request.head.ref }}" "origin/${{ github.event.pull_request.head.ref }}"; then
+          if git diff --quiet "origin/${{ github.event.pull_request.head.ref }}"; then
```

Comment:
so this is the main change?

Reply:
Yes and should be the only one. I opened this one but forgot to mention that it is not review-ready, I just needed CIs to run for validation.

Reply:
It is ready 👍🏼